source/hassio/installation.markdown

```diff
@@ -24,7 +24,7 @@ The following will take you through the steps required to install Home Assistant
 
 2. Install Home Assistant OS:
 
-   - **For the device images:** Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
+   - **For the device images:** Flash the downloaded image to a FAT formatted SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
```

Comment:
That actually doesn't matter. As the card is overwritten as a whole (including its current format).